packages/hash/frontend/src/blocks/page/createFormatPlugins/util.ts

```diff
@@ -13,6 +13,11 @@ export const selectionContainsText = (state: EditorState<Schema>) => {
     if (containsText) {
       return false;
     }
+
+    if (node.isInline) {
+      containsText = true;
+    }
```

Comment:
I'm not seeing what other inline node we would add that wouldn't contain text. Maybe images, but it would be hard to account for that now and we might as well just change it in the future if needed.